DASH-IF-IOP.bs.md

```diff
@@ -10,9 +10,1070 @@ Live discussion in #document-authoring on Slack.
+
+Note: To transform a sample timeline position `SampleTime` to an [=MPD timeline=] position, use the formula `MpdTime = Period@start + (SampleTime - @presentationTimeOffset) / @timescale`.
+
+`@presentationTimeOffset` SHALL NOT change between updates of a dynamic MPD.
+
+## Media segments ## {#timing-mediasegment}
+
+A <dfn>media segment</dfn> is an HTTP-addressable data structure that contains one or more media samples.
+
+[=Media segments=] SHALL contain one or more consecutive media samples. Consecutive [=media segments=] in the same [=representation=] SHALL contain consecutive media samples.
+
+[=Media segments=] SHALL contain the media samples that exactly match the time span on the [=sample timeline=] that is assigned to the [=media segment=] by the MPD, except when using [=simple addressing=] in which case a certain amount of inaccuracy MAY be present as defined in [[#timing-addressing-inaccuracy]].
+
+Advisement: Even when using [=simple addressing=], the [=media segment=] that starts at or overlaps the [=period=] start point SHALL contain a media sample that starts at or overlaps the [=period=] start point and the [=media segment=] that ends at or overlaps the [=period=] end point SHALL contain a media sample that ends at or overlaps the [=period=] end point.
+
+## Segment addressing modes ## {#timing-addressing}
+
+This section defines the <dfn title="addressing mode">addressing modes</dfn> that can be used for referencing [=media segments=] and initialization segments in interopreable DASH presentations.
+
+Addressing modes not defined in this chapter SHALL NOT be used by DASH services. Clients SHOULD support all addressing modes defined in this chapter.
+
+You SHOULD choose the addressing mode based on the nature of the content:
+
+<dl class="switch">
+
+: Content generated on the fly
+:: Use [=explicit addressing=].
+
+: Content generated in advance of publishing
+:: Use [=explicit addressing=] or [=indexed addressing=].
+
+</dl>
+
+A service MAY use [=simple addressing=] which enables the packager logic to be very simple. This simplicity comes at a cost of reduced applicability to multi-period scenarios and reduced client compatibility.
+
+All [=representations=] in the same adaptation set SHALL use the same addressing mode. [=Representations=] in different adaptation sets MAY use different addressing modes. [[#timing-connectivity|Period-connected representations]] SHALL use the same addressing mode in every [=period=].
+
+### Indexed addressing ### {#timing-addressing-indexed}
+
+A representation that uses <dfn>indexed addressing</dfn> consists of an [[!ISOBMFF]] track file containing an index segment, an initialization segment and a sequence of [=media segments=].
```

Comment:
From @haudiobe: This is not true. It contains a Segment Index, an Initialization Subsegment a sequence of Subsegments

Reply:
I agree that is the terminology used by MPEG-DASH. However, I reject that terminology as abominably confusing. Nobody calls them subsegments in the real world. MPEG needs to fix their terminology.